main.go

```diff
@@ -230,10 +229,8 @@ func handleError(err error) {
 	s := stderrRenderer.NewStyle().Padding(0, horizontalEdgePadding)
 	format := "\n%s\n\n"
 
-	var merr modsError
-	var ferr flagParseError
 	var args []interface{}
-	if errors.As(err, &ferr) {
+	if ferr, ok := err.(flagParseError); ok {
```

Comment:
why?

Reply:
it was broken before. We use a custom flag error handler https://github.com/charmbracelet/mods/pull/112/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R52-R54

Reply:
I mean, why changing the errors.As to the cast? I think both should work

Reply:
it doesn't matter for this one, but for the other error, we need to cast to be able to use `merr.reason`

Reply:
`errors.As` has the same effect, so it should have been working 🤔

https://pkg.go.dev/errors#As

Reply:
hmm it wasn't working with `modsError` yesterday 🤔

Reply:
ahh i think it wasn't working because of this https://github.com/charmbracelet/mods/pull/112/commits/e45b5aa54f8a9f62c37970afb4d1c1ef3f01c938#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261L99